.Lib9c.Tests/Action/RapidCombination6Test.cs

```diff
@@ -0,0 +1,722 @@
+                        gold = 0,
+                        actionPoint = 0,
+                        enhancementResult = ItemEnhancement9.EnhancementResult.GreatSuccess,
+                    };
+
+                    break;
+                }
+
+                case 10:
+                {
+                    Assert.True(ItemEnhancement10.TryGetRow(
+                        equipment,
+                        _tableSheets.EnhancementCostSheetV2,
+                        out var costRow));
+                    var equipmentResult = ItemEnhancement10.GetEnhancementResult(costRow, random);
+                    equipment.LevelUpV2(
+                        random,
+                        costRow,
+                        equipmentResult == ItemEnhancement10.EnhancementResult.GreatSuccess);
+                    resultModel = new ItemEnhancement10.ResultModel
+                    {
+                        id = mailId,
+                        preItemUsable = preItemUsable,
+                        itemUsable = equipment,
+                        materialItemIdList = new[] { materialEquipment.NonFungibleId },
+                        gold = 0,
+                        actionPoint = 0,
+                        enhancementResult = ItemEnhancement10.EnhancementResult.GreatSuccess,
+                    };
+
+                    break;
+                }
+
+                case 11:
+                {
+                    Assert.True(ItemEnhancement.TryGetRow(
+                        equipment,
+                        _tableSheets.EnhancementCostSheetV2,
+                        out var costRow));
+                    var equipmentResult = ItemEnhancement.GetEnhancementResult(costRow, random);
+                    equipment.LevelUpV2(
+                        random,
+                        costRow,
+                        equipmentResult == ItemEnhancement.EnhancementResult.GreatSuccess);
+                    resultModel = new ItemEnhancement.ResultModel
+                    {
+                        id = mailId,
+                        preItemUsable = preItemUsable,
+                        itemUsable = equipment,
+                        materialItemIdList = new[] { materialEquipment.NonFungibleId },
+                        gold = 0,
+                        actionPoint = 0,
+                        enhancementResult = ItemEnhancement.EnhancementResult.GreatSuccess,
+                        CRYSTAL = 0 * CrystalCalculator.CRYSTAL,
+                    };
+
+                    break;
+                }
+
+                default:
+                    break;
+            }
+
+            // NOTE: Do not update `mail`, because this test assumes that the `mail` was removed.
+            {
+                // var mail = new ItemEnhanceMail(resultModel, 0, random.GenerateRandomGuid(), requiredBlockIndex);
+                // avatarState.Update(mail);
+            }
```

Comment:
is it intended?

Reply:
Yes, I hope someone without this knowledge would know that not adding mail was intentional.